app/src/main/java/com/eveningoutpost/dexdrip/stats/FirstPageFragment.java

```diff
@@ -96,10 +110,10 @@ public void run() {
                 TextView medianView = (TextView) localView.findViewById(R.id.textView_median);
 
                 if (mgdl) {
-                    updateText(localView, medianView, Math.round(median * 10) / 10d + " mg/dl");
+                    updateText(localView, medianView, Math.round(median * 10) / 10d + " mg/dL");
```

Comment:
If the unit of measurements are correct here, shouldn't they fixed it anywhere? Otherwise, it is out of scope of this PR (which you stress often). I know that the wrong unit of measurement would be repeated in lines 76--80. This is unsatisfactory to me as well. I recommend to separate the change of the units at least into a separate commit.

Reply:
The units are correct.  There is just a typo.
There would be a discrepancy on the page if I used mmol/L for the new High and Low values but left mmol/l for mean and standard deviation on the same page.

Reply:
I meant "corrected". Fixing a typo is some kind of correction and a typo in a unit of measurement lead to an incorrect unit with respect to the value.
If "L" is used just on this page a discrepancy is introduced compared to the other places in xDrip were "mg/dl" or "mmol/l" is used. We should keep this in mind for a future PR, at least.

Reply:
I agree.  I would like all instances of units to be mg/dL and mom/L.
For some reason, we have mmol/l and mg/L in some places.  I would like to fix them all.
As you say, I would never have done this because it is a different fix.  I always make my PRs as small as possible.

The only reason I did it here is that otherwise, there would have been a discrepancy on the page.

In fact, I will be happy to search xDrip and open a PR correcting them all.

Reply:
Just for the future I recommend to copy & paste the erroneously written text, create the commit with the intentional change and fix the errors in a separate commit with it own descriptive commit message. Later on, we can identify it much easier based on the commit message.

Reply:
I agree.
I am going to change the units including the ones I am adding to all be incorrect but consistent.
I will open another PR correcting all instances in xDrip.